tests/test_wait_signal.py

```diff
@@ -335,7 +336,6 @@ class Obj(QtCore.QObject):
 
 
 class TestArgs:
-
```

Comment:
I don't really know what @nicoddemus uses - but either way I personally think doing cleanup changes as part of other PRs is problematic (when using git blame, when having to revert something, etc.). I think @nicoddemus is a bit more relaxed than I in that regard though :laughing: